index.css

```diff
@@ -0,0 +1,145 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+
+/*begin custom*/
+
+.topBorderSpacing {
+  height: 64px;
+}
+
+.canvasContainer {
+	box-sizing: content-box;
+  /*width: 768px;*/
+  /*height: 512px;*/
+  width: 768px;
+  height: 512px;
+  margin: auto;
+}
+
+.pixelRow {
+	display: flex;
+}
+
+.pixel {
+	box-sizing: border-box;
+	border-top: 1px;
+  border-left: 1px;
+  border-style: solid;
+  border-color: white;
+  width: 16px;
+  height: 16px;
+  background-color: #F2F2F2;
+}
+
+.canvasContainer .pixelRow:nth-child(1n) .pixel:nth-child(48) {
+  border-right: 1px;
+  border-style: solid;
+  border-color: white;
+}
+
+.canvasContainer .pixelRow:nth-child(32) .pixel:nth-child(1n+1) {
+  border-bottom: 1px;
+  /*find out why the style and color props are not inherited from cascade*/
+  border-style: solid;
+  border-color: white;
+}
+
+.paletteSuperContainer {
+	display: flex;
+}
+
+.swatchContainer {
+	margin: auto;
+	height: 48px;
+	display: flex;
+	align-items: center;
+}
+
+.paletteSwatch {
+	box-sizing: border-box;
+	width: 24px;
+	height: 24px;
+	border-radius: 50%;
+}
+
+.paletteSwatchPadding {
+	display: inline-block;
+	padding: 2px;
+  margin: 4px 4px 4px 4px;
+}
+
+.paletteSelectedSwatchStyle {
+	border: 2px solid white;
+	border-radius: 50%;
+}
+
+.paletteSwatchColor1 {
```

Comment:
`paletteSwatchColor` is not a good name for this class, it is not descriptive. In this case, `red` would have been a better choice.

This comment also applies to all the `paletteSwatchColor`'s below.